apps/bfd-model/bfd-model-rif/src/main/resources/db/migration/V66__new_schema_snf_claims_fix.sql

```diff
@@ -0,0 +1,522 @@
+                    icd_prcdr_cd4,
+                    icd_prcdr_cd5,
+                    icd_prcdr_cd6,
+                    icd_prcdr_cd7,
+                    icd_prcdr_cd8,
+                    icd_prcdr_cd9,
+                    icd_prcdr_cd10,
+                    icd_prcdr_cd11,
+                    icd_prcdr_cd12,
+                    icd_prcdr_cd13,
+                    icd_prcdr_cd14,
+                    icd_prcdr_cd15,
+                    icd_prcdr_cd16,
+                    icd_prcdr_cd17,
+                    icd_prcdr_cd18,
+                    icd_prcdr_cd19,
+                    icd_prcdr_cd20,
+                    icd_prcdr_cd21,
+                    icd_prcdr_cd22,
+                    icd_prcdr_cd23,
+                    icd_prcdr_cd24,
+                    icd_prcdr_cd25,
+                    icd_prcdr_vrsn_cd1,
+                    icd_prcdr_vrsn_cd2,
+                    icd_prcdr_vrsn_cd3,
+                    icd_prcdr_vrsn_cd4,
+                    icd_prcdr_vrsn_cd5,
+                    icd_prcdr_vrsn_cd6,
+                    icd_prcdr_vrsn_cd7,
+                    icd_prcdr_vrsn_cd8,
+                    icd_prcdr_vrsn_cd9,
+                    icd_prcdr_vrsn_cd10,
+                    icd_prcdr_vrsn_cd11,
+                    icd_prcdr_vrsn_cd12,
+                    icd_prcdr_vrsn_cd13,
+                    icd_prcdr_vrsn_cd14,
+                    icd_prcdr_vrsn_cd15,
+                    icd_prcdr_vrsn_cd16,
+                    icd_prcdr_vrsn_cd17,
+                    icd_prcdr_vrsn_cd18,
+                    icd_prcdr_vrsn_cd19,
+                    icd_prcdr_vrsn_cd20,
+                    icd_prcdr_vrsn_cd21,
+                    icd_prcdr_vrsn_cd22,
+                    icd_prcdr_vrsn_cd23,
+                    icd_prcdr_vrsn_cd24,
+                    icd_prcdr_vrsn_cd25,
+                    prcdr_dt1,
+                    prcdr_dt2,
+                    prcdr_dt3,
+                    prcdr_dt4,
+                    prcdr_dt5,
+                    prcdr_dt6,
+                    prcdr_dt7,
+                    prcdr_dt8,
+                    prcdr_dt9,
+                    prcdr_dt10,
+                    prcdr_dt11,
+                    prcdr_dt12,
+                    prcdr_dt13,
+                    prcdr_dt14,
+                    prcdr_dt15,
+                    prcdr_dt16,
+                    prcdr_dt17,
+                    prcdr_dt18,
+                    prcdr_dt19,
+                    prcdr_dt20,
+                    prcdr_dt21,
+                    prcdr_dt22,
+                    prcdr_dt23,
+                    prcdr_dt24,
+                    prcdr_dt25 )
+select
+${logic.psql-only}  cast(clm_id as bigint),
+${logic.psql-only}  cast(bene_id as bigint),
+${logic.psql-only}  cast(clm_grp_id as bigint),  
+${logic.hsql-only}  convert(clm_id, SQL_BIGINT),
+${logic.hsql-only}  convert(bene_id, SQL_BIGINT),
+${logic.hsql-only}  convert(clm_grp_id, SQL_BIGINT),
```

Comment:
The psql/hsql for same columns are all psql then all hsql here but below they are psql/hsql for each column.  Might be better to follow the same pattern in both places.

Reply:
yea, was torn between having all HSQL and PSQL or having the table columns in order; changed to all HSQl/PSQL together. thx.